docs/reference/src/main/asciidoc/environments.asciidoc

```diff
@@ -708,13 +708,17 @@ In this case, Weld will use a "flat" deployment structure - all bean classes sha
 
 NOTE: Bean archive isolation is supported (and enabled by default) from version 2.2.0.Final. Previous versions only operated with the "flat" deployment structure.
 
+TIP: All Weld SE specific configuration properties could be also set through CDI 2.0 API, i.e. using `SeContainerInitializer.addProperty()` and `SeContainerInitializer.setProperties()` methods.
+
 ==== Implicit Bean Archive Support
 
 CDI 1.1 introduced the bean discovery mode of `annotated` used for implicit bean archives (see also <<packaging-and-deployment>>). This mode may bring additional overhead during container bootstrap.
 Therefore, Weld Servlet supports the use of https://github.com/wildfly/jandex[Jandex] bytecode scanning library to speed up the scanning process. Simply put the http://search.maven.org/#search|gav|1|g%3A%22org.jboss%22%20AND%20a%3A%22jandex%22[jandex.jar] on the classpath.
 If Jandex is not found on the classpath Weld will use the Java Reflection as a fallback.
 
-In general, an implicit bean archive does not have to contain a beans.xml descriptor. However, such a bean archive is not supported by Weld SE, i.e. it's excluded from discovery.
+By default, an implicit bean archive that does not contain a beans.xml descriptor is excluded from discovery.
+However, it is possible to intruct Weld to scan all class path entries - either set the system property `org.jboss.weld.se.scan.classpath.entries`/`javax.enterprise.inject.scan.implicit` to `true` or use `Weld.property()` and `SeContainerInitializer.addProperty()` methods.
+In this case, such implicit bean archives are also discovered.
```

Comment:
If you use the suggested edit above, then this sentence could be:
`Using any of the options above will result in implicit bean archives being discovered.`

Reply:
Will do, thanks!